ProQuest.js

```diff
@@ -449,13 +471,15 @@ function scrape(doc, url, type, pdfUrl) {
 			mimeType: 'application/pdf'
 		});
 	} else {
-		var pdfLink = ZU.xpath(doc, '(//div[@id="side_panel"]//\
+		var pdfLink = ZU.xpath(doc, '//li[@id="tab-SPDF-null" or @id="tab-FulltextPreview-null"]/a');
```

Comment:
(well, one reason might be that there are still pages that use the old layout, but I'm thinking the answer is no. In that case, we should get rid of any old code that is no longer relevant)